tests/cmake_integration/run_ci_tests.pl

```diff
@@ -47,12 +48,14 @@ ($)
     "compiler=s" => \$compiler,
     "skip-run-test" => \$skip_run_test,
     "skip-cxx11" => \$skip_cxx11,
+    "skip-typecode" => \$skip_typecode,
     "no-shared" => \$no_shared,
     );
-
-my @dirs = ('../Nested_IDL', 'Messenger_1', 'Messenger_2');
-push @dirs, '../generated_global' unless $no_shared;
-push @dirs, 'C++11_Messenger' unless $skip_cxx11;
+my @dirs;
+#my @dirs = ('../Nested_IDL', 'Messenger_1', 'Messenger_2');
+#push @dirs, '../generated_global' unless $no_shared;
+#push @dirs, 'C++11_Messenger' unless $skip_cxx11;
```

Comment:
Why is this changing?

Reply:
Sorry, forgot to clean that up from testing.